Modelica/Resources/C-Sources/zlib/gzlib.c

```diff
@@ -1,11 +1,11 @@
 /* gzlib.c -- zlib functions common to reading and writing gzip files
- * Copyright (C) 2004-2017 Mark Adler
+ * Copyright (C) 2004-2019 Mark Adler
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
 
 #include "gzguts.h"
 
-#if defined(_WIN32) && !defined(__BORLANDC__) && !defined(__MINGW32__)
+#if defined(_WIN32) && !defined(__BORLANDC__)
```

Comment:
I was a bit surprised by this removal of MINGW32 (added in the previous version), but digging more I see that it is due to https://github.com/madler/zlib/commit/a3622937453f517285f851817a7505f54f4f19aa and thus ok.